.github/workflows/linux_unit_tests_with_latest_deps.yml

```diff
@@ -83,4 +83,5 @@ jobs:
         name: Upload coverage to Codecov
         uses: codecov/codecov-action@v1
         with:
+          token: ${{ secrets.CODECOV_TOKEN }}
```

Comment:
So something changed with the `codecov/codecov-action@v1` action which now requires this token, whereas previously we didn't have to provide it?

Reply:
Correct me if I'm wrong, but the @v1 stuff refers to which version of codecov-action we're checking out, right?  I think, if you're interested in previous versions of codecov action, you can change the 1 to a 2 or also specify the specific version of codecov-action you want to try to see whether the latest requires the token or not.  Still, very weird that it doesn't correspond to a recent release.

Reply:
I'm not so sure that it's an issue with the codecov action, it doesn't seem to have released anything recently. It might be related to how secrets are accessed by GitHub Actions?